indico/modules/receipts/default_templates/attendance/metadata.yaml

```diff
@@ -0,0 +1,83 @@
+custom_fields:
+  - name: address_from
+    type: textarea
+    attributes:
+      label: Organizer Address
+
+  - name: venue
+    type: input
+    attributes:
+      label: Venue
+
+  - name: logo_url
+    type: image
+    attributes:
+      label: Logo
+
+  - name: add_url
+    type: checkbox
+    attributes:
+      label: Add event URL?
+
+  - name: text
+    type: textarea
+    attributes:
+      label: Certificate Text
+      value: >-
+        The Organizing Comitee certifies that {person} ({person_affiliation}) participated in the event {event}, which took place {date}, at {venue}.
+      placeholders:
+        person: Full name of the participant
+        person_affiliation: Participant's affiliation
+        event: Title of the event
+        date: Start date of the event
```

Comment:
Should we also add one for the end date?
for "participated in the training/workshop/whatever from xxx to xxx" etc